index.md

```diff
@@ -13,5 +13,8 @@ description: "NIST Special Publication 800-63 Digital Identity Guidelines"
 
 In addition to introducing the detailed guidelines in these areas, SP 800-63-3 addresses the factors involved in choosing the appropriate Identity Assurance Level (IAL), Authentication Assurance Level (AAL), and Federation Assurance Level (FAL) for a given application.
 
-These implementation resources are provided pursuant to OMB memorandum [M-19-17](https://www.whitehouse.gov/wp-content/uploads/2019/05/M-19-17.pdf). While these resources reference normative guidelines in the SP 800-63-3 document suite and other documents, these resources themselves are not normative. Comments on these guidelines are welcomed and can be submitted via email to [dig-comments@nist.gov](mailto:dig-comments@nist.gov).
+These implementation resources are provided pursuant to OMB memorandum [M-19-17](https://www.whitehouse.gov/wp-content/uploads/2019/05/M-19-17.pdf). While these resources reference normative guidelines in the SP 800-63-3 document suite and other documents, these resources themselves are not normative. 
+These implementation resources provide guidance for SP 800-63-3 in three parts: Part A addresses SP 800-63A, Part B addresses SP 800-63B, and Part C addresses SP 800-63C. Section numbers are presented in parentheses in each part which refer to the SP 800-63-3 volume corresponding to that part. 
```

Comment:
I don't think we followed the parenthesis aspect throughout the document. In truth, I think we should go back and add actual links to the right parts of the document wherever possible. That's what we tend to do with the FAQs. That can be a clean up step as it doesn't affect the content.